Drop tools (and related tests) as they rely on Python 2 / pythoncard
--- libkate-0.4.1.orig/Makefile.am
+++ libkate-0.4.1/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS=. tools tests doc
+SUBDIRS=. doc
 
 OGGDIR=built-streams
 kateincdir=$(includedir)/kate
